client/src/components/OverView/ImageGallery.jsx

```diff
@@ -39,7 +43,7 @@ const MainGallery = styled(Swiper)`
     display: flex;
     justify-content: center;
     overflow: hidden;
-    background: #EEEEEE;
+    /* background: #EEEEEE; */
```

Comment:
Remember to clean this up later if you're not using the formatting

Reply:
will do!